sybil/parsers/codeblock.py

```diff
@@ -46,7 +63,7 @@ def __call__(self, document):
             # There must be a nicer way to get code.co_firstlineno
             # to be correct...
             line_count = document.text.count('\n', 0, source_start)
-            if self.future_imports:
+            if language == 'python' and self.future_imports:
```

Comment:
This is a bit unpleasant... I wonder if we can find a more graceful way?

Reply:
I didn't want to break the existing interface or existing code that uses this class.

Reply:
I'm happy to do this as a new major release, in the face of other languages, it should be:

```python
class CodeBlockParser:
    
    def __init__(self, language, evaluator=None):
        ...
        
class PythonCodeBlockParser(CodeBlockParser):

    language = 'python'
    
    def __init__(self, future_imports=...):
        ...
        
    def evaluator(...):
        pass
```